Editor/Window/ConnectToFleetInput.cs

```diff
@@ -108,41 +115,27 @@ private void AssignUiElements(VisualElement container)
             _fleetConnectFoldout = container.Q("AnywherePageConnectFleetTitle");
             _cancelButton = container.Q<Button>("AnywherePageCreateFleetCancelButton");
         }
-        
-        private async Task SetupFleetMenu()
-        {
-            await UpdateFleetMenu();
-            _fleetNameDropdownContainer.RegisterValueChangedCallback(_ =>
-                {
-                    _stateManager.SelectedFleetName = _fleetNameDropdownContainer.value;
-                    var currentFleet = _fleetsList.First(fleet => fleet.Name == _fleetNameDropdownContainer.value);
-                    _fleetIdText.text = currentFleet.FleetId;
-                    FleetId = currentFleet.FleetId;
-                    _stateManager.CoreApi.PutSetting(SettingsKeys.FleetName, currentFleet.Name);
-                }
-            );
-        }
-        
+
         private async Task UpdateFleetMenu()
         {
             if (_stateManager.GameLiftWrapper != null)
             {
                 _fleetsList = await _fleetManager.ListFleetAttributes();
-                s_fleetNameList.Clear();
-                _fleetsList.ForEach(fleet => s_fleetNameList.Add(fleet.Name));
-                _fleetNameDropdownContainer.choices = s_fleetNameList;
+                _fleetNameDropdownContainer.choices = _fleetsList.Select(fleet => fleet.Name).ToList();
+                _fleetNameDropdownContainer.value = _stateManager.AnywhereFleetName;
+                _fleetIdText.text = _stateManager.AnywhereFleetId;
             }
         }
-        
+
         private async void SetupBootMenu()
```

Comment:
minor: what exactly is the `BootMenu`? If there's a better naming possible for this, please update this

Reply:
renamed `SetupPage`